### PR TITLE
nsjail: update to 2.5, don't git clone, fix musl, install manpage

### PR DESCRIPTION
**Pull request by mvf**

### CODE REVIEW

File: srcpkgs/nsjail/template [hunk 1]
```diff
@@ -1,28 +1,36 @@
 # Template file for 'nsjail'
 pkgname=nsjail
-version=2.4
-revision=2
+version=2.5
+revision=1
 build_style=gnu-makefile
-hostmakedepends="libnl3-devel git pkg-config protobuf-c bison flex"
-makedepends="libnl3-devel libcap-devel protobuf-devel"
+hostmakedepends="bison flex pkg-config protobuf-devel"
```
Comment (cr6git) [inline srcpkgs/nsjail/template]: You can omit `bison` which is a dependency of `flex`.

File: srcpkgs/nsjail/template [hunk 2]
```diff
@@ -1,28 +1,36 @@
...
+makedepends="libnl3-devel protobuf-devel"
 short_desc="A light-weight process isolation tool"
 maintainer="Andrea Brancaleoni <abc@pompel.me>"
 license="Apache-2.0"
 homepage="https://github.com/google/nsjail"
```
Comment (cr6git) [inline srcpkgs/nsjail/template]: `homepage="http://nsjail.com"`

Comment (mvf) [reply srcpkgs/nsjail/template]: Good point, updated.

File: srcpkgs/nsjail/template (hunk 1 above)
Comment (mvf) [reply srcpkgs/nsjail/template]: IMHO that's an implementation detail. The build process calls both, so it should depend on both. `flex` (the program) doesn't really depend on `bison` either, I don't know why the Void package does. Looks like a bug to me.

Comment (cr6git) [reply srcpkgs/nsjail/template]: If details are not important it's okay...

Comment (cr6git) [reply srcpkgs/nsjail/template]: I'm probably wrong. Make a PR to change this then! :+1:

**Comment by maxice8** [comment]
2.6 is available.

**Pull request closed without merging**